### ISSUE TITLE
Running the machine

### ISSUE THREAD

**Issue by kandjgameing824** [devos50/qemu#64]
when i use the command to run it from the paths to the images i keep running into this error zsh: parse error near `\n' if anyone can help thanks!

**Comment by devos50** [comment]
Thanks for reporting - could you provide the full command you're trying to run the emulator with?

**Comment by kandjgameing824** [comment]
Thanks for the response! i used the command you provided in the guide here ./arm-softmmu/qemu-system-arm -M iPod-Touch,bootrom=<path to bootrom image>,iboot=<path to iboot image>,nand=<path to nand directory> -serial mon:stdio -cpu max -m 1G -d unimp -pflash <path to NOR image> i as well use apple silicon if that means anything thanks again!

**Comment by devos50** [comment]
```
./arm-softmmu/qemu-system-arm -M iPod-Touch,bootrom=,iboot=,nand= -serial mon:stdio -cpu max -m 1G -d unimp -pflash
```

Are you sure that you are providing the paths to the bootrom, iBoot and NAND images?

**Comment by kandjgameing824** [comment]
yes i provided all paths here is the command with that ./arm-softmmu/qemu-system-arm -M iPod-Touch,bootrom=/Users/james/Downloads/bootrom_s5l8900,iboot=/Users/james/Downloads/iboot_204_n45ap.bin,nand=/Users/james/Downloads/iboot_204_n45ap.bin -serial mon:stdio -cpu max -m 1G -d unimp -pflash this seems to not have the same error as when i previously tryed this but now it returns a differnt error zsh: no such file or directory: ./arm-softmmu/qemu-system-arm Thanks for the quick response!

**Comment by devos50** [comment]
Are you sure that the compilation process finished without any error? This might prohibit the `qemu-system-arm` binary from being generated.

**Comment by kandjgameing824** [comment]
ive attempted to recompile the qemu verison and now ran into another error ERROR: "cc" cannot build an executable (is your linker broken?)

make: *** No targets specified and no makefile found.  Stop.
i have ninja and gcc  as well as x code installed

**Comment by NateTheAppleLover** [comment]
I had the exact same problem, after also having troble with Ninja.

**Comment by devos50** [comment]
@kandjgameing824 which platform/OS are you trying to compile on? Do you have additional log output as well?

**Comment by spetterman66** [comment]
"i have ninja and gcc as well as x code installed"

Sounds like macOS.
________________________________
